### DISCUSSION TITLE
Can Popover be used inside a modal dialog?

### DISCUSSION THREAD

**Discussion by nicholasrice-hs** [adobe/react-spectrum#7016] [Q&A]
I'm trying to use components implementing Popover inside a dialog, such as `Select` and `DatePicker`, however elements in the flyout seem to never be able to receive focus. Ideally, I'd like to leverage the `<dialog>` element because my browser matrix supports it, however [elements inside the Popover don't seem to be able to receive focus](https://stackblitz.com/edit/vitejs-vite-zd5tcw?file=src%2FApp.tsx). Next, I tried using `react-aria-components` [modal / dialog pattern](https://react-spectrum.adobe.com/react-aria/Dialog.html) but I am [running into the same issue.](https://stackblitz.com/edit/vitejs-vite-3mw7av?file=src%2FApp.tsx&terminal=dev)

Is there a way to make these Popover controls work inside modal dialogs?

**Comment by snowystinger** [comment]
Looks like it works following the examples on our docs
https://stackblitz.com/edit/vitejs-vite-46c7ne?file=src%2FApp.tsx,src%2Findex.css&terminal=dev
(apologies for the bad styling, I put that together quickly)

There's also an example on https://react-spectrum.adobe.com/react-aria/index.html if you follow
"Modal" on that very first example. There are pickers and date pickers in there.
